app/tokenHelper.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const fs = require('fs');
+
+const tokenLocation = "slack.token";
+
+if (!fs.existsSync(tokenLocation)) { // eslint-disable-line node/no-deprecated-api
+  fs.openSync(tokenLocation, 'w');
+}
```

Comment:
From the documentation:
Note that `fs.exists()` is deprecated, but `fs.existsSync()` is not. (The callback parameter to `fs.exists()` accepts parameters that are inconsistent with other Node.js callbacks. `fs.existsSync()` does not use a callback.)